openapi/paths/objects@{object_id}@access@{access_id}.yaml

```diff
@@ -26,4 +56,19 @@ get:
       $ref: '../components/responses/500InternalServerError.yaml'
   tags:
     - Objects
-  x-swagger-router-controller: ga4gh.drs.server
\ No newline at end of file
+  x-swagger-router-controller: ga4gh.drs.server
+  parameters:
+    - $ref: '../components/parameters/ObjectId.yaml'
+    - $ref: '../components/parameters/AccessId.yaml'
+  requestBody:
+    required: true
+    content:
+      multipart/form-data:
+        schema:
+          type: object
+          properties:
+            tokens[]:
```

Comment:
See comment about using `tokens` instead of `tokens[]`

Reply:
done